src/commandList.cpp

```diff
@@ -303,1138 +305,1130 @@ void CommandList::initNular(::sqf::runtime::runtime& runtime) {
     runtime.register_sqfop(nular("isstreamfriendlyuienabled", "", [](sqf::runtime::runtime& runtime) -> value { runtime.__logmsg(logmessage::runtime::ErrorMessage(runtime.context_active().current_frame().diag_info_from_position(), "NOT IMPLEMENTED", "isstreamfriendlyuienabled")); return {}; }));
     runtime.register_sqfop(nular("confignull", "", [](sqf::runtime::runtime& runtime) -> value { runtime.__logmsg(logmessage::runtime::ErrorMessage(runtime.context_active().current_frame().diag_info_from_position(), "NOT IMPLEMENTED", "confignull")); return {}; }));
     runtime.register_sqfop(nular("get3deniconsvisible", "", [](sqf::runtime::runtime& runtime) -> value { runtime.__logmsg(logmessage::runtime::ErrorMessage(runtime.context_active().current_frame().diag_info_from_position(), "NOT IMPLEMENTED", "get3deniconsvisible")); return {}; }));
-
-
-
-
-
-
-
-
-
 }
 
 
 void CommandList::initBinary(::sqf::runtime::runtime& runtime) {
 
-    runtime.register_sqfop(binary(4,"lnbsetcurselrow", t_any(), t_any(), "", [](sqf::runtime::runtime& runtime, value::cref l, value::cref r) -> value { runtime.__logmsg(logmessage::runtime::ErrorMessage(runtime.context_active().current_frame().diag_info_from_position(), "NOT IMPLEMENTED", "lnbsetcurselrow")); return {}; }));
```

Comment:
Whitespace change messes everything up, I'd prefer a simpler diff over fixing the missing whitespace